www/class/centreonAuth.class.php

```diff
@@ -181,7 +173,7 @@ protected function checkPassword($password, $token = "", $autoimport = false)
             }
 
             foreach ($authResources as $arId) {
```

Comment:
Don't forget to create a db transaction when there is more than one update request. (=> foreach)